### PR TITLE
Banner Tracks delete message

### PR DESCRIPTION
**Pull request by brianteeman**
This PR changes the message when there are no tracks to delete from “0 records are deleted" to “No Tracks to Delete.”

See #4765

### CODE REVIEW

File: administrator/components/com_banners/controllers/tracks.php [hunk 1]
```diff
@@ -84,10 +84,13 @@ public function delete()
 		{
 			JError::raiseWarning(500, $model->getError());
 		}
-		else
+		else if(count > 0)
```
Comment (zero-24) [inline administrator/components/com_banners/controllers/tracks.php]: @brianteeman can you chage it to

`elseif (count > 0)`

so we have the space between the `elseif` and the opening `(` :smile: Thanks

File: administrator/components/com_banners/controllers/tracks.php [hunk 2]
```diff
@@ -84,10 +84,13 @@ public function delete()
...
 		{
 			$this->setMessage(JText::plural('COM_BANNERS_TRACKS_N_ITEMS_DELETED', $count));
 		}
+		else{
```
Comment (zero-24) [inline administrator/components/com_banners/controllers/tracks.php]: can you move the `{` into the next line?

File: administrator/language/en-GB/en-GB.com_banners.ini [hunk 3]
```diff
@@ -211,6 +211,7 @@ COM_BANNERS_SUBMENU_TRACKS="Tracks"
 COM_BANNERS_TRACKS_DELETE="Delete Tracks"
 COM_BANNERS_TRACKS_DOWNLOAD="Download tracks"
 COM_BANNERS_TRACKS_EXPORT="Export"
+COM_BANNERS_TRACKS_0_ITEMS_DELETED = "No Tracks to Delete."
```
Comment (zero-24) [inline administrator/language/en-GB/en-GB.com_banners.ini]: i guess we don't need the spaces bevor and after the `=`

**Comment by zero-24** [comment]
@test successfull. Thanks Brian!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6871">issues.joomla.org/joomla-cms/6871</a>.</sub>

File: administrator/components/com_banners/controllers/tracks.php [hunk 4]
```diff
@@ -84,10 +84,14 @@ public function delete()
 		{
 			JError::raiseWarning(500, $model->getError());
 		}
-		else
+		elseif(count > 0)
```
Comment (zero-24) [inline administrator/components/com_banners/controllers/tracks.php]: one space between `elseif` and the opening `(` please :smile:

File: administrator/components/com_banners/controllers/tracks.php [hunk 5]
```diff
@@ -84,10 +84,14 @@ public function delete()
...
 			JError::raiseWarning(500, $model->getError());
 		}
-		else
+		elseif (count > 0)
 		{
 			$this->setMessage(JText::plural('COM_BANNERS_TRACKS_N_ITEMS_DELETED', $count));
 		}
+		else
+		{
+			$this->setMessage(JText::plural('COM_BANNERS_TRACKS_0_ITEMS_DELETED'));
```
Comment (joomdonation) [inline administrator/components/com_banners/controllers/tracks.php]: @brianteeman Please change thí line to 
````php
$this->setMessage(JText::_('COM_BANNERS_TRACKS_0_ITEMS_DELETED'));
````
Just use JText::_ instead using JText::plural.

After that change, @test success from me.

**Comment by infograf768** [comment]
Please change also the constant to:
`COM_BANNERS_TRACKS_NO_ITEMS_DELETED` instead of `0`

**Comment by Dylis** [comment]
@test - Patch OK (I didnt make any change on constant) - No Tracks to Delete. is correctly displayed.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6871">issues.joomla.org/joomla-cms/6871</a>.</sub>

**Comment by brianteeman** [comment]
@infograf768 change made

**Comment by Kubik-Rubik** [comment]
Thank you @brianteeman for the contribution!

**Pull request closed without merging**